docs/dev-setup/devnet-setup.md

```diff
@@ -7,7 +7,7 @@ See [Logging Control](logging-control.md) for information on controlling
 logging output from the `obc-peer` and chaincodes.
 
 
-**Note:** When running with security enabled, follow the security setup instructions described in [Chaincode Development](https://github.com/openblockchain/obc-docs/blob/master/api/SandboxSetup.md#security-setup-optional) to set up the CA server and log in registered users before sending chaincode transactions.
+**Note:** When running with security enabled, follow the security setup instructions described in [Chaincode Development](https://github.com/openblockchain/obc-docs/blob/master/api/SandboxSetup.md#security-setup-optional) to set up the CA server and log in registered users before sending chaincode transactions.  In this case peers started using Docker images need to point to the correct CA address (default is localhost). CA addresses have to be specified in openchain.yaml variables paddr of eca, tca and tlsca.
```

Comment:
Should change file path:
https://github.com/openblockchain/obc-docs/blob/master/api/SandboxSetup.md#security-setup-optional
->
https://github.com/hyperledger/fabric/blob/master/docs/API/SandboxSetup.md#security-setup-optional

And change the file name:
openchain.yaml
->
core.yaml